src/SourceGenerators/Uno.UI.Tasks/XamlGenerator/BackingFieldDefinition.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace Uno.UI.SourceGenerators.XamlGenerator
+{
+	internal class BackingFieldDefinition
+	{
+		public BackingFieldDefinition(string type, string name, Accessibility accessibility)
+		{
+			Type = type;
+			Name = name;
+			Accessibility = accessibility;
+		}
+
+		public string Name { get; }
+		public object Type { get; }
```

Comment:
Should it stay a `string` ?

Reply:
This fie wasn't supposed to be part of the PR. Fixed.